### ISSUE TITLE
Add  culling to GH backup CI of superset dashboards

### ISSUE THREAD

**Issue by mrjones-plip** [medic/cht-sync#173]
**What feature do you want to improve?**
We offer a way to to automate backup of superset dashboards.  however, [the CI job we](https://github.com/medic/cht-sync/tree/main/.github/actions/superset-backup) run will fill up with a backup every day.  While handy - this is likely too many files which will never get used.

**Describe the improvement you'd like**
The CI job should not only make a backup, but also cull older backups from the GH repo.  Something like keeping 1/day x 7 days, 1/week x 4 weeks, 1/month x 12 months.

**Describe alternatives you've considered**
manually cull or do nothing at all

**Additional context**
we're hoping to have national deployments run this and a more clean list of files in backup dir will be much easier when it comes time to restore.

**Comment by dianabarsan** [comment]
@mrjones-plip the CI job can be setup to run on whatever schedule maintainers decide, it doesn't have to be daily. I think automatic deletions are quite dangerous.

**Comment by mrjones-plip** [comment]
Fair point!  While think that the old adage  "backup is only as good as the restore" is very applicable here, I can agree that some docs on how to cull older backups would suffice!
